server/arweave/router.ts

```diff
@@ -117,7 +117,6 @@ router.post('/upload',
         arweaveId,
         list,
       },
-      ,
       numAssetIds,
     ] = await Promise.all(promises)
```

Comment:
promises in order would be `uploadFilesToArweave`, and `uploadFilesToIPFS` and possibly `registerNUMAssets(` in the third element, so that comma should be needed and not an error?

Reply:
I think adding `_` to indicate unused param for the 2nd promise is a good idea though